src/components/ImportParameters.jsx

```diff
@@ -0,0 +1,158 @@
+  },
+  http_get_parameters: {
+    name: <><code>http_get_parameters</code></>, 
+    type: "map with single values or tuples of values", 
+    description: "The map will be flattened into pairs that are appended to the IRI before making an HTTP request" 
+  },
+  http_post_parameters: {
+    name: <><code>http_post_parameters</code></>, 
+    type: "map with single values or tuples of values", 
+    description: "The map will be flattened into pairs that are sent as the body of an HTTP POST request" 
+  },
+  iri_fragment: {
+    name: <><code>iri_fragment</code></>, 
+    type: "String", 
+    description: <>add a fragment to a <code>resource</code> or <code>endpoint</code> IRI</>
+  },
+  base: {
+    name: <><code>base</code></>, 
+    type: "String", 
+    description: "specify the base IRI to be used when importing RDF data; if given, relative IRIs will be made absolute based on this base; otherwise, relative IRIs remain relative in Nemo"
+  },
+  endpoint: {
+    name: <><code>endpoint</code></>, 
+    type: "IRI, String", 
+    description: "Any endpoint that implements the W3C SPARQL standard." 
+  },
+  query: {
+    name: <><code>query</code></>, 
+    type: "String", 
+    description: "The SPARQL query sent to the endpoint. The query will be validated before sending. Make sure to include all necessary prefixes." 
+  },
+};
+
+const parameters = {
+  csv: ["resource","format","compression","limit","ignore_headers","http_headers","http_get_parameters","http_post_parameters","iri_fragment"],
+  dsv: ["resource","format","compression","limit","ignore_headers","delimiter","http_headers","http_get_parameters","http_post_parameters","iri_fragment"],
+  //tsv: ["resource","format","compression","limit","ignore_headers","http_headers","http_get_parameters","http_post_parameters","iri_fragment"],
+  rdf: ["resource","format","compression","limit","base","http_headers","http_get_parameters","http_post_parameters","iri_fragment"],
+  //nquads: ["resource","format","compression","limit","base","http_headers","http_get_parameters","http_post_parameters","iri_fragment"],
+  //trig: ["resource","format","compression","limit","base","http_headers","http_get_parameters","http_post_parameters","iri_fragment"],
+  //ntriples: ["resource","format","compression","limit","base","http_headers","http_get_parameters","http_post_parameters","iri_fragment"],
+  //rdfxml: ["resource","format","compression","limit","base","http_headers","http_get_parameters","http_post_parameters","iri_fragment"],
+  //turtle: ["resource","format","compression","limit","base","http_headers","http_get_parameters","http_post_parameters","iri_fragment"],
+  json: ["resource","format","compression","http_headers","http_get_parameters","http_post_parameters","iri_fragment"],
+  sparql: ["endpoint","query","http_headers","http_get_parameters","http_post_parameters","iri_fragment"],
+
+};
+
+
+export default function InteractiveTable() {
```

Comment:
This also ties the (in-principle reusable) component `InteractiveTable` to the import parameter data. If at all, the data should be provided via Props, so that we can indeed reuse the table elsewhere.